routing_common/transit_types.hpp

```diff
@@ -30,6 +30,12 @@ ShapeId constexpr kInvalidShapeId = std::numeric_limits<ShapeId>::max();
 // To convert double to uint32_t at better accuracy |kInvalidWeight| should be close to real weight.
 Weight constexpr kInvalidWeight = -1.0;
 
+#define DECLARE_TRANSIT_TYPE_FRIENDS                                                           \
```

Comment:
Strongly suggest you to undef the macro at the end of the header.

Reply:
Done